src/components/Expense.js

```diff
@@ -10,10 +13,15 @@ export default function Expense({
   paidBy,
   paidFor,
 }) {
+  function handleClick() {
+    setEditExpense(expense);
+    console.log(expense);
+  }
+
   return (
-    <Wrapper>
+    <Wrapper to="/edit" onClick={handleClick}>
```

Comment:
I would recommend against using a `Link` as a wrapper and mixing it with an `onClick` event. I would suggest to use `navigate` from the `useNavigate` (or so) hook from `react-router-dom` to navigate when the function is called.

Also, a `button` with a descriptive text ("Edit Expense") would semantically be more correct.

```js
const navigate = useNavigate()

function handleClick() {
    setEditExpense(expense);
    navigate("/edit")
  }

  return (
    <Wrapper onClick={handleClick}>
```